demos/smart_classroom_demo/cpp/src/cnn.cpp

```diff
@@ -99,20 +105,18 @@ void VectorCNN::Compute(const std::vector<cv::Mat>& images, std::vector<cv::Mat>
         return;
     }
     vectors->clear();
-    auto results_fetcher = [vectors, outp_shape](const InferenceEngine::BlobMap& outputs, size_t batch_size) {
+    auto results_fetcher = [vectors, outp_shape](const std::map<std::string, ov::runtime::Tensor>& outputs, size_t batch_size) {
         for (auto&& item : outputs) {
-            InferenceEngine::Blob::Ptr blob = item.second;
-            if (blob == nullptr) {
+            ov::runtime::Tensor blob = item.second;
+            if (!blob) {
```

Comment:
That's actually strange to expect tensor to be empty. Please remove the check and rename `blob` to `tensor` or `out`

Reply:
Done